### DISCUSSION TITLE
デザインのコンセプトについて

### DISCUSSION THREAD

**Discussion by kagomen** [lgtm-factory/lgtm-factory#32] [未分類]
LGTM Factoryのサイトのデザインを作成するにあたって、コンセプトを明確にします！
以下は @kazzyfrog さんのディスコードの文章ですー

> - サービスの内容：
LGTM画像生成サービス
> - 前提：
「LGTM」は、チーム開発で、コードレビューOK（承認）の際に使われる開発者の慣習的な言葉。
「LGTM」という言葉だけでもいいけど、「LGTM という言葉入りの画像」を貼る慣習も存在する！
なので、いくつか「LGTM 画像サービス」というのは、既に存在している
> - 概要：
> GitHub 上での、テキストコミュニケーションを、明るくカジュアルにするための、簡単に使える「LGTM 画像サービス」を作るプロジェクト。
> - 既存の「LGTM 画像サービス」との違い：
秩序なくアップロードされた画像に、「LGTM」のワードをはめるよりは、もっとリッチなデザインにしたい
なので、実際の画像ではなく、HTML、CSS（またはJSX）で、書いたコード（デザインしたUI）から画像を生成する。（もちろん画像も使用可能）
APIでは、URLクエリパラメータを動的にして、それに応じた画像を生成し、返却したい。
例：「abc.com/sticker1?message=have-a-nice-day」にアクセスしたら、「HAVE A NICE DAY」というメッセージ入りのLGTM画像が生成される
誰でもプロジェクトの参加（コントリビュート）できて、デザインを追加できる

また、現時点では、サイトの雰囲気の案として、以下が挙がっています

- ゲームっぽい感じ
- 工場っぽい感じ
- カントリー系
- マイクラっぽい感じ
- ピコピコ系

**Comment by kagomen** [comment]
プロジェクト名が LGTM Factory なので、「工場」とプロジェクトの概要を良い感じにかけ合わせて、一言でプロジェクトについて表現できると良いかな？という感じがします！　それがコンセプトになるかな？と

**Reply by kazzyfrog** [reply]
> 一言でプロジェクトについて表現

いいですね！！👍

一旦、仕様を決めてからの方が、こちらの議論も進みやすそうなので、
仕様が決まったら、書き込んでいきますー！

**Reply by kagomen** [reply]
了解です！

**Comment by kazzyfrog** [comment]
## デザイン
アイデア段階ですが、ざっくり書いてきます！
必要があれば、コメントを追加したり、編集を加えてください！👍

要点を整理したのち、
一言でプロジェクトについて表現できるコンセプトキーワードを設定する！

**Reply by kazzyfrog** [reply]
下記全て、アイデア段階。暫定。。
## 要点整理
**「LGTM　＋　Factory」**

サイト（ブランド）のイメージ・世界観：
- フォーマルというよりは、インフォーマル。つまりカジュアル
- 開発者、インターネットの文化が入っている感じ
- 類似のLGTMサービスと違い、適当な画像（時に不適切と思われる画像）がアップされていない。つまりある程度管理されている。
- ごちゃごちゃ画像がある昔のサイト、、っていうよりは、最近のデザインツール的な、シンプルさ？統一感？がある。
- 開発者が、HTMLを書いてデザインを追加できる、OSSのコミュニティ感・一体感
- 工場っぽい
- →工場というより、図工って感じかも？文化祭？、、〇〇ミュージアムみたいな感じ？？
- カントリー系
- ピコピコ系
- ゲーム系なら、ネオンカラー？テクノ？レトロ？近未来？ドット？サイバーパンク？ゆるキャラ？
- ゲームセンター感？溜まり場感？基地感？工場感？ガレージでのDIY？

連想するもの・キーワード：
- ゲーム・マイクラ
- LINEスタンプ
- ステッカー（PCに貼るような？）
- キットカット（一言メッセージ）

**Reply by kagomen** [reply]
> キットカット（一言メッセージ）

これ、良い・・・！

**Reply by siso001** [reply]
図工のテーマ良さそう！
定規とダンボールを連想する
![images (7).jpeg](https://github.com/user-attachments/assets/30703c28-3688-438f-b269-c3d162776069)

![images (9).jpeg](https://github.com/user-attachments/assets/6960aab5-5432-47bd-b41b-8ef4aefa2dfe)

**Reply by kagomen** [reply]
クラフト、ワーク系ですね！ワクワクさん的な・・・！👀

**Comment by kazzyfrog** [comment]
## その後：
コンセプトキーワードが決まったら、
イメージソースを収集し、ムードボードを作成する！

これはFigmaで、コンセプトキーワードに沿った画像素材・スクショなどを、ペタペタ貼っていく感じかな。
テキストから、視覚的な認識合わせ！

---

そして、
- ざっくりブランドカラー・使用するカラーの決定（カラーパレットも？）
- ロゴ系
- デザインカンプ作成

こんな感じかな。。

**Comment by kagomen** [comment]
こちら https://github.com/lgtm-factory/lgtm-factory/discussions/31#discussioncomment-10299547 を読んでて1つアイデアが思い浮かんだので、共有します！！

- プロジェクト名: LGTM Lab
  - もしくはLGTM Labo,  LGTM Lab., LGTM研究室
- コンセプト:
  - タイトルそのままですが、「LGTM研究室：LGTMからリモートワークでのチームの在り方を考える」みたいな感じ？
- 雰囲気:
  - 実験室みたいな感じ
  - 🧪🥼🔬

---

と、書いてて気付きましたが、なんか骨太すぎて、今回のような単発プロジェクト向きなコンセプトではないかも・・・？💧

**Reply by kagomen** [reply]
でも、もし骨太OK・長期OKなら、プロジェクトでNoteアカウント作ってコラムとか書いたりしたら、サービスとしてすごく魅力的だなーと思いました！

**Reply by kazzyfrog** [reply]
確かに、研究室・実験室的な方向もいいですね！
拡張性ありそうな感じというか、クリエイティブな感じ！

> 骨太OK・長期OK

これは全然、８月で１回区切りながらも、
その後もメンテしたい人で、メンテしていく形になると思うので、問題ないかと思います！

自分も、LGTM Factoryが取り組む領域に関して、なんか書こうかと思ってました！

**Reply by kagomen** [reply]
> これは全然、８月で１回区切りながらも、
その後もメンテしたい人で、メンテしていく形になると思うので、問題ないかと思います！

たしかに、了解です！
